### PR TITLE
Add toast notifications

### PR DESCRIPTION
**Pull request by mwodahl**
# PR Details
## Description

This PR adds toast notifications to the admin menus, the contact support menu, and the snmp menus. Additionally, some of the success/error state logic has been removed as it is no longer needed with the toast notifications. 

## How Has This Been Tested?

This was tested locally via docker-compose and was verified to work with the admin add/edit/delete menus, the contact support menu, and the snmp menus. 

## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [ X ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] My changes require new environment variables:
  - [ ] I have updated the docker-compose, K8s YAML, and all dependent deployment configuration files.
- [ ] My changes require updates to the documentation:
  - [ ] I have updated the documentation accordingly.
- [ x ] My changes require updates and/or additions to the unit tests:
  - [ x ] I have modified/added tests to cover my changes.
- [ x ] All existing tests pass.

### CODE REVIEW

**Comment by drewjj** [comment]
The toast notifications look really nice. It looks like they may not be entirely accurate in some cases. A good example is when trying to delete a SNMP config when it is running locally. The API fails to really delete it but the toast notification says it was successful. Not sure if the API is just not returning a good failure case but just wanted to point this out.

image

**Comment by drewjj** [comment]
I am getting this error message when I try to delete a RSU on the admin page. However the API does delete it. Seems to be a web app handling issue.

Cannot read properties of undefined (reading 'success')
TypeError: Cannot read properties of undefined (reading 'success')
    at http://localhost:3000/static/js/bundle.js:10101:20

**Comment by drewjj** [comment]
The users adding/deleting works great.

**Comment by mwodahl** [comment]
> I am getting this error message when I try to delete a RSU on the admin page. However the API does delete it. Seems to be a web app handling issue.
> 
> Cannot read properties of undefined (reading 'success') TypeError: Cannot read properties of undefined (reading 'success') at http://localhost:3000/static/js/bundle.js:10101:20

I was able to replicate this and it looks like the API was failing to delete an RSU as there were still records tied to it in the public.snmp_msgfwd_config table. This was also occurring in dev so I updated the API to delete those records and now this error should no longer be present.

**Comment by mwodahl** [comment]
> The toast notifications look really nice. It looks like they may not be entirely accurate in some cases. A good example is when trying to delete a SNMP config when it is running locally. The API fails to really delete it but the toast notification says it was successful. Not sure if the API is just not returning a good failure case but just wanted to point this out.
> 
> image

It looks like the API is currently returning a 200 code regardless of the SNMP command status which is why this is showing a successful delete. Would it interfere with anything to refactor the API to return a different code when it fails to perform an SNMP command?

File: webapp/src/features/adminEditUser/adminEditUserSlice.tsx [hunk 1]
```diff
@@ -130,10 +126,14 @@ export const submitForm = createAsyncThunk(
...
-      return false
+      let res = await dispatch(editUser({ json: tempData }))
+      if ((res.payload as any).success) {
+        return { submitAttempt: false, success: true, message: 'User Updated Successfully' }
+      } else {
+        return { submitAttempt: false, success: false, message: (res.payload as any).message }
+      }
     } else {
-      return true
+      return { submitAttempt: true, success: false, message: 'Please fill out all required fields' }
```
Comment (dmccoystephenson) [inline webapp/src/features/adminEditUser/adminEditUserSlice.tsx:136]: What is the purpose of this `submitAttempt` variable? Does it refer to whether the user has attempted to submit the form? Renaming the variable to be more descriptive may be helpful here.

**Review by dmccoystephenson — CHANGES_REQUESTED** [review]
I am running into failures when executing the webapp unit tests:

image

Looks like the snapshots need to be updated

**Review by drewjj — APPROVED** [review]
I think we can move forward with this how it is. If the SNMP always gives a 200 that is fine for now I suppose. We may want to consider changing that in the future to prevent confusion.

**Pull request merged**